plugins/CoreHome/vue/src/Alert/alert.less

```diff
@@ -14,14 +14,26 @@
   border-radius: 2px;
   font-size: 14px;
   position: relative;
-  &:before {
+
+  @media (max-width: 600px) {
+    padding: 15px;
+  }
+
+  &::before {
     font-family: "matomo";
     content: "\e88f";
     position: absolute;
     left: 20px;
     line-height: 100%; // line-height = font-size
     font-size: 24px;
-    margin-top: -3px
+    margin-top: -3px;
+
+    @media (max-width: 600px) {
+      position: static;
+      display: block;
+      margin-bottom: 0.5em;
+      text-align: center;
+    }
```

Comment:
ℹ️ Make the icon on top instead on the left of the message on mobile. It will take less space.